src/main/java/com/questhelper/helpers/quests/theheartofdarkness/TheHeartOfDarkness.java

```diff
@@ -0,0 +1,1028 @@
+        inBossRoom = new ZoneRequirement(bossRoom);
+
+        pulledFirstLever = new VarbitRequirement(11138, 1);
+        pulledSecondLever = new VarbitRequirement(11139, 1);
+        // on first frozen platform:
+        // 11181 0->1
+        pulledThirdLever = new VarbitRequirement(11137, 1);
+//        pulledFourthLever = new VarbitRequirement(11136, 1);
+
+        unlockedShortcut = new VarbitRequirement(11174, 1);
+
+        inspectedAirMarkings = new ManualRequirement();
+        inspectedEarthMarkings = new ManualRequirement();
+        inspectedWaterMarkings = new ManualRequirement();
+        inspectedFireMarkings = new ManualRequirement();
+
+        repairedWaterStatue = new VarbitRequirement(11127, 1, Operation.GREATER_EQUAL);
+        repairedFireStatue = new VarbitRequirement(11124, 1, Operation.GREATER_EQUAL);
+        repairedEarthStatue = new VarbitRequirement(11126, 1, Operation.GREATER_EQUAL);
+        repairedAirStatue = new VarbitRequirement(11125, 1, Operation.GREATER_EQUAL);
+
+        takenOrUsedAirIcon = or(repairedAirStatue, airIcon);
+        takenOrUsedEarthIcon = or(repairedEarthStatue, earthIcon);
+        takenOrUsedFireIcon = or(repairedFireStatue, fireIcon);
+        takenOrUsedWaterIcon = or(repairedWaterStatue, waterIcon);
+
+        activatedFirstStatue = new VarbitRequirement(11161, -1);
+        activatedSecondStatue = new VarbitRequirement(11162, -1);
+        activatedThirdStatue = new VarbitRequirement(11163, -1);
+        activatedFourthStatue = new VarbitRequirement(11164, -1);
+
+        activateStatueRequirements[0] = activatedFirstStatue;
+        activateStatueRequirements[1] = activatedSecondStatue;
+        activateStatueRequirements[2] = activatedThirdStatue;
+        activateStatueRequirements[3] = activatedFourthStatue;
+
+        // 1651, 9641, 0
+        elementalStatueDetails = new ArrayList<>();
+        elementalStatueDetails.add(new StatueDetails(new Zone(new WorldPoint(1648, 9620, 0), new WorldPoint(1653, 9624, 0)), inspectedEarthMarkings,
+                new WorldPoint(1605, 9635, 0),"earth"));
+        elementalStatueDetails.add(new StatueDetails(new Zone(new WorldPoint(1610, 9639, 0), new WorldPoint(1615, 9643, 0)), inspectedFireMarkings,
+                new WorldPoint(1605, 9627, 0), "fire"));
+        elementalStatueDetails.add(new StatueDetails(new Zone(new WorldPoint(1610, 9619, 0), new WorldPoint(1616, 9624, 0)), inspectedWaterMarkings,
+                new WorldPoint(1608, 9624, 0), "water"));
+        elementalStatueDetails.add(new StatueDetails(new Zone(new WorldPoint(1648, 9639, 0), new WorldPoint(1653, 9642, 0)), inspectedAirMarkings,
+                new WorldPoint(1608, 9638, 0), "air"));
+        // Failed puzzle
+        // 11132 0->1
+    }
+
+    private void setupSteps()
+    {
+        talkToItzlaAtTeomat = new NpcStep(this, NpcID.PRINCE_ITZLA_ARKAN_12896, new WorldPoint(1454, 3173, 0), "Talk to Prince Itzla Arkan at the Teomat. You" +
```

Comment:
would be nice with a widget highlight for the "The Teomat" widget in the quetzal interface. 874.15[1] iirc, model ID 51205 if we can match on model ID since it'd be a bit more stable across updates.